src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TemplateRenderer.cs

```diff
@@ -172,14 +175,31 @@ private IEnumerable<string> GetViewNames()
             {
                 if (typeof(IEnumerable).IsAssignableFrom(fieldType))
                 {
+                    if (typeof(IEnumerable<IFormFile>).IsAssignableFrom(fieldType))
```

Comment:
You mean do this? https://github.com/aspnet/Mvc/commit/4558a333dca48212d685d07c7469fd442ab8d4a2#diff-2f73dc4b7cc96312c3a65b0c24251c9dR178

I had this in my previous commit. I changed it after discussing with @dougbu regarding this comment https://github.com/aspnet/Mvc/pull/2366#discussion_r28095216